core/exception/ExceptionHandler.ts

```diff
@@ -370,16 +370,22 @@ export class ExceptionHandler {
 
         const ret = {};
         forOwn(obj, (val, key) => {
-            if (key.startsWith('_')) return;
-            if (val && !val.toJSON) {
-                if (isObject(val)) {
-                    val = depth > 1 ? this.cloneAndTrim(val, depth - 1) : '{...}';
-                }
-                if (isArray(val)) {
-                    val = depth > 1 ? val.map(it => this.cloneAndTrim(it, depth - 1)) : '[...]';
+            try {
+                if (key.startsWith('_')) return;
+                if (val && !val.toJSON) {
+                    if (isObject(val)) {
+                        val = depth > 1 ? this.cloneAndTrim(val, depth - 1) : '{...}';
+                    }
+                    if (isArray(val)) {
+                        val = depth > 1 ? val.map(it => this.cloneAndTrim(it, depth - 1)) : '[...]';
+                    }
                 }
+                ret[key] = val;
+            } catch (e) {
+                // fail quietly
+                // some properties may be inaccessible - eg: security
```

Comment:
This comment looks incomplete - is there a specific instance we want to point future-us towards here? Wasn't sure what to make of `eg. security` - sounds like something specific :)